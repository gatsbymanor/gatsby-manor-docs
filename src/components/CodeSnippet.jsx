import React from 'react';

const CodeSnippet = (props) => {
  const { children } = props;

  return (
    <pre><code>
      {children}
    </code></pre>
  );
}

export default CodeSnippet;
