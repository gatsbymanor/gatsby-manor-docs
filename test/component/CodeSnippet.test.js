import React from 'react';
import CodeSnippet from '../../src/components/CodeSnippet';

describe('<CodeSnippet />', () => {

  const component = mount(
    <CodeSnippet />
  );

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });
});
