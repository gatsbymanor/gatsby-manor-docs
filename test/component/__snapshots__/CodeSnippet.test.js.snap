// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CodeSnippet /> should render correctly 1`] = `
<CodeSnippet>
  <pre>
    <code />
  </pre>
</CodeSnippet>
`;
